.gitignore

```diff
@@ -13,6 +13,7 @@ Homestead.yaml
 npm-debug.log
 yarn-error.log
 .phpunit.result.cache
+phpunit.xml
```

Comment:
let's keep `phpunit.xml` in git, it's easier when cloning and just `./vendor/bin/phpunit` wdyt? let's try to setup sqlite in-memory for tests in `phpunit.xml` too

Reply:
PhpUnit will use `phpunit.xml` _and_ `phpunit.xml.dist` if present (in that order). This means you can override any test-configuration you want in your own `phpunit.xml` file, but you can also just use the default `phpunit.xml.dist`. 

[Here is a StackOverflow answer from the creator of PhpUnit explaining exactly this](https://stackoverflow.com/a/44764078/4795870).

I mostly prefer to use the same database in testing and in production (in this case MySQL), to prevent some oddities from slipping through. I've encountered issues that would show up in SQLite (in the tests), but not when using MySQL (in development/staging/production).